app/vili.go

```diff
@@ -159,7 +159,7 @@ func New() *App {
 					registryID = &ecrAccountID
 				}
 				err := repository.InitECR(&repository.ECRConfig{
-					Region:          config.GetString(config.AWSRegion),
+					Region:          config.GetString(config.ECRRegion),
```

Comment:
Should this fall back to `AWSRegion` if blank?

Reply:
Great point!

I usually think that explicit configuration is best, so I made the `ecr-region`  a required configuration variable.  Let me know what you think..... if this doesn't make sense, happy to change this to fall back to using the `aws-region` instead.

Reply:
After an end of the day conversation with @BrianBland yesterday, we decided to change the logic to default to using the `aws-region` variable if `ecr-region` is not set. 

I've added logic to make this so.